Windows 17 fails on extensions/vertx-http/deployment module and thus there are many other modules skipped

`2024-01-04T08:28:05.2884931Z [ERROR] io.quarkus.vertx.http.http2.Http2RSTFloodProtectionTest.testRstFloodProtectionWithTlsEnabled  Time elapsed: 10.009 s  <<< FAILURE!`

There are also some strange failures of `io.quarkus.vertx.http.testrunner` tests, e.g. `io.quarkus.vertx.http.testrunner.SimpleET`. But they are not reported in the final summary.

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -23,7 +23,7 @@
         <jandex.version>3.1.2</jandex.version>
         <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
         <javax.inject.version>1</javax.inject.version>
-        <parsson.version>1.1.2</parsson.version>
+        <parsson.version>1.1.5</parsson.version>
         <resteasy-microprofile.version>2.1.1.Final</resteasy-microprofile.version>
         <resteasy-spring-web.version>3.0.2.Final</resteasy-spring-web.version>
         <resteasy.version>6.2.4.Final</resteasy.version>

--- a/independent-projects/resteasy-reactive/pom.xml
+++ b/independent-projects/resteasy-reactive/pom.xml
@@ -41,7 +41,7 @@
         <jakarta.json-api.version>2.1.2</jakarta.json-api.version>
         <jakarta.ws.rs-api.version>3.1.0</jakarta.ws.rs-api.version>
         <jakarta.xml.bind-api.version>4.0.0</jakarta.xml.bind-api.version>
-        <parsson.version>1.1.2</parsson.version>
+        <parsson.version>1.1.5</parsson.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.target>11</maven.compiler.target>
         <maven.compiler.source>11</maven.compiler.source>

